spec/services/proofing/lexis_nexis/ddp/verification_request_spec.rb

```diff
@@ -20,6 +20,7 @@
       email: 'test@example.com',
       request_ip: '127.0.0.1',
       uuid_prefix: 'ABCD',
+      uuid: '1234',
```

Comment:
Minor: I might suggest using a (faked) valid UUIDv4 here so that it's at least the same format as we expect to send, for clearer expectation-setting like what we do with other values in this payload.

e.g.

```suggestion
      uuid: '00000000-0000-0000-0000-000000000000',
```